edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Hosting.cs

```diff
@@ -42,12 +44,25 @@ public class Hosting
                             port,
                             listenOptions =>
                             {
-                                listenOptions.UseHttpsExtensions(
+                                listenOptions.UseHttps(
                                     new HttpsConnectionAdapterOptions()
                                     {
                                         ServerCertificate = serverCertificate,
-                                        ClientCertificateValidation = (clientCert, chain, policyErrors) => true,
-                                        ClientCertificateMode = certificateMode,
+                                        OnAuthenticate = (context, options) =>
```

Comment:
This is a bit confusing, but is the best solution we could find to save the client's cert chain. See [issue](https://github.com/dotnet/aspnetcore/issues/21606) for context.